File: BookCheck/src/main/java/com/bespalaya/bookCheck/config/MyAbstractAnnotationConfigDispatcherServletInitializer.java
package com.bespalaya.bookCheck.config;

import org.springframework.web.servlet.support.AbstractAnnotationConfigDispatcherServletInitializer;

public class MyAbstractAnnotationConfigDispatcherServletInitializer extends AbstractAnnotationConfigDispatcherServletInitializer {
    protected Class<?>[] getRootConfigClasses() {
        return null;
    }

    protected Class<?>[] getServletConfigClasses() {
        return new Class[] {WebConfig.class};
    }

    protected String[] getServletMappings() {
        return new String[] {"/"};
    }
}

File: laba_chetyre/src/main/kotlin/by/bsut/labachetyre/config/MyAbstractAnnotationConfigDispatcherServletInitializer.kt
package by.bsut.labachetyre.config

import org.springframework.web.servlet.support.AbstractAnnotationConfigDispatcherServletInitializer

class MyAbstractAnnotationConfigDispatcherServletInitializer : AbstractAnnotationConfigDispatcherServletInitializer() {
    protected val rootConfigClasses: Array<Any>? = null
    protected val servletConfigClasses: Array<Any> = arrayOf(SpringWebConfig::class.java)
    protected val servletMappings: Array<String> = arrayOf("/")
}